FrontEnd/src/components/Admin/OrthancSettings.js

```diff
@@ -24,6 +25,13 @@ export default class OrthancSettings extends Component {
     async componentDidMount(){
         let answer = await apis.options.getOrthancServer()
         this.setState(answer)
+        let verbosity = await apis.options.getVerbosity()
```

Comment:
Tu a dejà la fonction en this.getVerbosity qui fait ca, peut etre mieux de l'appeler plutot que de dupliquer l'appel au repository d'API ?
@beber32